README.md

```diff
@@ -21,6 +21,7 @@ npm install gulp-mocha
 npm install babel-register
 npm install express
 npm install body-parser
+npm install nodemon
```

Comment:
Collapse all of these `npm install X` bits into one `npm install`.

When running `npm install` without arguments, it will install all of the dependencies specified in the `package.json`.

Reply:
@lewismc (ping)

Reply:
@rgfoley do you want to include them in parent README or examples one?

Reply:
@lewismc Include them in neither. If you look in the `package.json` file, you'll see all those dependencies are listed. If you clone the repo and run `npm install` with no arguments, all of the dependencies will get installed, no need to run each one individually.

Might be explaining this incorrectly, but the behavior is there. Does that make sense?

Reply:
ACK, update made. thank you